code-studio/src/js/progress.js

```diff
@@ -160,12 +166,61 @@ progress.renderStageProgress = function (stageData, progressData, clientProgress
 };
 
 progress.renderCourseProgress = function (scriptData) {
-  var teacherCourse = $('#landingpage').hasClass('teacher-course');
+  var store = loadProgress(scriptData);
   var mountPoint = document.createElement('div');
+
+  $.ajax('/api/user_progress/' + scriptData.name).done(function (data) {
```

Comment:
An arrow function would be appropriate here, if your'e trying to move this file to ES6.